Plots/Panels/NCL_panel_35.py

```diff
@@ -0,0 +1,240 @@
+      print("generate_2d_array: the list of positions for the highs must be the same size as num_high.")
+
+#
+#  Dims are reversed in order to get the same results as the NCL function.
+#
+  nx = int(dims[1])
+  ny = int(dims[0])
+  out_array = np.zeros([nx,ny],'f')
+  tmp_array = np.zeros([3,51],'f')
+  fovm = 9./float(nx)
+  fovn = 9./float(ny)
+  nlow = max(1,min(25,num_low))
+  nhgh = max(1,min(25,num_high))
+  ncnt = nlow + nhgh
+
+#
+  for k in range(num_low):
+    if not lows_at is None:
+      tmp_array[0,k] =  float(lows_at[k][1])   # lows at specified locations.
+      tmp_array[1,k] =  float(lows_at[k][0])
+      tmp_array[2,k] = -1.
+    else:
+      tmp_array[0,k] =  1.+(float(nx)-1.)*_dfran() # lows at random locations.
+      tmp_array[1,k] =  1.+(float(ny)-1.)*_dfran() # lows at random locations.
+      tmp_array[2,k] = -1.
+  for k in range(num_low,num_low+num_high):
+    if not highs_at is None:
+      tmp_array[0,k] =  float(highs_at[k-num_low][1])  # highs locations
+      tmp_array[1,k] =  float(highs_at[k-num_low][0])  # highs locations
+      tmp_array[2,k] =  1.
+    else:
+      tmp_array[0,k] =  1.+(float(nx)-1.)*_dfran() # highs at random locations.
+      tmp_array[1,k] =  1.+(float(ny)-1.)*_dfran() # highs at random locations.
+      tmp_array[2,k] =  1.
+  
+  dmin =  1.e+36
+  dmax = -1.e+36
+  midpt = 0.5*(minv + maxv)
+  for j in range(ny):
+    for i in range(nx):
+      out_array[i,j] = midpt
+      for k in range(ncnt):
+        tempi = fovm*(float(i+1)-tmp_array[0,k])
+        tempj = fovn*(float(j+1)-tmp_array[1,k])
+        temp  = -(tempi*tempi + tempj*tempj)
+        if (temp >= -20.):
+          out_array[i,j] = out_array[i,j] +    \
+             midpt*tmp_array[2,k]*math.exp(temp)
+      dmin = min(dmin,out_array[i,j])
+      dmax = max(dmax,out_array[i,j])
+                   
+  out_array = (((out_array-dmin)/(dmax-dmin))*(maxv-minv))+minv
+
+  del tmp_array
+
+  return np.transpose(out_array,[1,0])
+
+
+def _get_double(obj,name):
+  return(NhlGetDouble(_int_id(obj),name))
+
+def _get_double_array(obj,name):
+  return(NhlGetDoubleArray(_int_id(obj),name))
+
+###############################################################################
+# Create dummy data
+nx = 100
+ny = 100
+data1 = generate_2d_array((ny,nx), 10, 10, -19., 16., 0)
+data2 = generate_2d_array((ny,nx), 10, 10, -28., 15., 1)
+data3 = generate_2d_array((ny,nx), 10, 10, -25., 18., 2)
+
+###############################################################################
+# Create figure and axes using gvutil
+fig, axs = plt.subplots(1, 3, figsize=(12, 6), sharex='all', sharey='all', gridspec_kw={'wspace': 0})
+
+gvutil.set_axes_limits_and_ticks(axs[0], xticks=np.arange(0, 120, 20), yticks=np.arange(0, 120, 20), xticklabels=np.arange(0, 100, 20), yticklabels=np.arange(0, 100, 20))
+gvutil.add_major_minor_ticks(axs[0], x_minor_per_major=4, y_minor_per_major=4)
+axs[0].tick_params(axis='both', which='both', top=True, left=True, right=False)
```

Comment:
Fyi: Setting `top=True` might not be needed because it is already set `True` in `gvutil.add_major_minor_ticks()`